sklearn/neural_network/tests/test_mlp.py

```diff
@@ -432,7 +432,7 @@ def test_predict_proba_binary():
     X = X_digits_binary[:50]
     y = y_digits_binary[:50]
 
-    clf = MLPClassifier(hidden_layer_sizes=5)
+    clf = MLPClassifier(hidden_layer_sizes=5, activation = 'logistic')
```

Comment:
don't use space between operators for keyword arguments:

`activation='logistic'`

Reply:
okay I will change that.
Changing the activation function doesn't guarantee the model will converge. Does the test need a better input restriction or should it do something on convergence warning?

Reply:
According to my previous code snippet using activation='logistic' passes 100 random seeds so that seem to solve the convergence issue.

I think we should also remove 

`with ignore_warnings(category=ConvergenceWarning): `